CHANGELOG.md

```diff
@@ -291,4 +291,16 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - `hostrule` with `sslKeyCertificate` of type `secret` can now be configured in all namespaces in OpenShift clusters.
  - Fixed an issue of VS creation failing with 470 Ingress with a single path giving a 470 error.
- - `nodeNetworkList` is not mandatory for NSX-T Overlay deployments.
\ No newline at end of file
+ - `nodeNetworkList` is not mandatory for NSX-T Overlay deployments.
+
+ ## AKO-1.8.2
+
+### Changed
+- Logs, from Istio file watcher, are changed from `Error` level to `Warn` level.
+- Status of ingress/route will be populated with IPV6 address in IPV6 deployments instead of IPV4 ips.
```

Comment:
please change it to `IPv6` and `IPv4`.

Reply:
when IPV6 deployment is there, IPV6 will get preference over IPV4. So only IPV6 addresses will be populated in status. cc: @aaha97